### PR TITLE
UI of input box improved

### LINKED ISSUE #124: Reduce distance between messages and input box in dark mode
Reduce the distance between messages and input box in dark mode to Improve UI. 

2021-03-28_LI

### PR DESCRIPTION
**Pull request by akshay1027**
UI of the input box is improved. Fixes #124.

2021-03-31 (1)_LI

### CODE REVIEW

**Comment by akshay1027** [comment]
necessary changes have been made!

**Comment by dilemmawalker** [comment]
Hey @DhairyaBahl , please have a look to verify final changes.

**Comment by DhairyaBahl** [comment]
@akshay1027 Kindly squash all the commits ! Rest seems alright to me !

File: src/App.js [hunk 1]
```diff
@@ -65,16 +65,6 @@ function App() {
...
-  let but;
-  but = (
-    <input
-      className={`input ${dark ? "dark_input" : "light_input"}`}
-      placeholder="Write Your Message"
-      value={input}
-      onChange={(event) => setInput(event.target.value)}
-    />
-  );
-
```
Comment (DhairyaBahl) [inline src/App.js:77]: You deleted this ! Is the app still working ?? Do send the gif or video for the same.

File: src/App.js [hunk 2]
```diff
@@ -150,19 +140,6 @@ function App() {
...
-                <Button
-                  className="iconButton"
-                  onClick={newMessage}
-                  type="submit"
-                  variant="contained"
-                >
-                  {" "}
-                  <SendIcon />
-                </Button>
-              </form>
```
Comment (DhairyaBahl) [inline src/App.js:165]: Same goes with this one ! Kindly confirm it once with gif and screenshots !!

**Review by DhairyaBahl — CHANGES_REQUESTED** [review]

**Comment by akshay1027** [comment]
yep it is working! It is the second input box which i deleted before..!

**Comment by akshay1027** [comment]
https://drive.google.com/file/d/1QuNNAtDxow1hVa04lT_-YtfFIIgmqrka/view?usp=sharing , @DhairyaBahl  please check out this video! Website is working well! please check your mail as well!

**Comment by akshay1027** [comment]
can you please add labels to this PR, Thank you! @DhairyaBahl @dilemmawalker

**Comment by dilemmawalker** [comment]
@akshay1027 please squash all the commits & then we can merge your pull request again.

**Pull request merged**